docs/source/screening.rst

```diff
@@ -3,7 +3,7 @@ Screening
 
 .. note::
 
-  Only for Oracle and Exploration projects. Read more about :ref:`project_create:Project modes`.
+  Only for Oracle and Validation. Read more about :doc:`simulation_overview`.
```

Comment:
Is this rendered as Overview or Simulation Overview? I prefer the reference to Project Modes.

Reply:
In project_mode, the simulation mode is not described, therefore I changed the reference to where the simulation mode is described. 

(it is rendered as `Overview`, I updated the text